### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by developertariq**
### ✔Built only below pages:
     The home page.
     The about page.
## ✔This project is developed to fulfill the following objectives: 
- Use semantic HTML tags.
- Apply best practices in HTML code.
- Use CSS selectors correctly.
- Use CSS box model.
- Use Flexbox to place elements in the page.
- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
- Use GitHub Pages to deploy web pages.
- Apply JavaScript best practices and language style guides in code.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Use objects to store and access data.
- Communicate technical concepts to other technical people.

### CODE REVIEW

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @developertariq , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

almost-there-reach


## Highlights
All linters are passing :heavy_check_mark: 
Followed GitHub flow :heavy_check_mark: 
Nice code structure :heavy_check_mark: 



## Required Changes ♻️

- [ ] Your video is missing highlight about a piece of code that you find very interesting or is very proud of. Kindly update the project documentation video and add that. :wink: 

- [ ] Kindly ensure that the top dark nav bar contains the logout and language links in addition to the social media links to match the design. 

 Your design            |  The  design 
:-------------------------:|:-------------------------:
Screenshot from 2022-11-24 13-31-00  |  Screenshot from 2022-11-24 13-35-14


- [ ] Please note that the desktop version footer of the about page has a dark background instead of the white background you are currently using. Moreover, there is no footer for the mobile version of the  home page. Kindly check and fix these to match the design. :wink: 

- [ ] Kindly ensure that all images have alt attributes and their appropriate values in conformity with best standard practices. :pray: Some of your images do not have empty alt attribute values as shown below 

Screenshot from 2022-11-24 13-47-43


- [ ] Kindly make sure that your css stylesheet is DRY in order to conform with best practices. I see a lot of repetions. For instance, you have repeated the same code `@media screen and (min-width: 768px) ` 6 times in your main.css file and 3 times in your about.css file

- [ ] Kindly ensure that the pages use the exact same colors for text, backgrounds, borders etc from the original design. I can see you are using `#ef513c` for `#ec5242` and `#272a31` instead of `#d3d3d3` for some of the text and background. Please consider going through your code to correct this anomaly. 

 Your design            |  The  design 
:-------------------------:|:-------------------------:
Screenshot from 2022-11-24 14-01-50  |  Screenshot from 2022-11-24 14-01-34

 Your design            |  The design 
:-------------------------:|:-------------------------:
Screenshot from 2022-11-24 14-05-29  |  Screenshot from 2022-11-24 14-05-35


- [ ] Kindly ensure that all pages have similar spacing, shadows, and paddings as well as margins as the original design. The margins and paddings of the home and about page is different from the design. Kindly check and fix this. 

 Your design            |  The design 
:-------------------------:|:-------------------------:
Screenshot from 2022-11-24 14-08-27  |  Screenshot from 2022-11-24 14-09-12




_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Hello @developertariq, :wave: Kindly ensure that all your changes for the project are present in the final submission branch. :pray:  Currently it looks like you have merge and deleted some branches before submitting the last one for review which does not make your work look proffessional. It also makes it difficult for reviewers to check and review your code :tired_face: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @developertariq, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- All linters passed ✔️
- Home and About pages 👍🏼 
- Updated Readme ✔️

 ## Required Changes ♻️
_Check the comments under the review._
- [x] As the previous reviewer pointed out, the video presentation is still missing the aspect that interests you. I will encourage you to specifically state that 'this this this' are the areas that interest me.
- [x] The changes you made are not reflected on your published pages, kindly ensure all changes are published accordingly so that they won't only be in your GitHub account but also seen live.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/about.css [hunk 1]
```diff
@@ -35,6 +35,7 @@
   padding: 20px 20px;
   background-color: #fff;
   border: 1px solid #d3d3d3;
+  font-size: 0.875em;
```
Comment (christianbiring1) [inline css/about.css:38]: - [ ] As said previously, Kindly make sure you implement the suggested changes by the previous reviewers.You need to stick to the origin design `(typos, font, spacing, and color)`

What was said previously :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @developertariq , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS: REQUEST CHANGES :recycle: 
soclose



 ## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Desscriptive PR `title` and `summary`

 ## Required Changes ♻️
- [ ] `KINDLY MAKE SURE YOU IMPLEMENT ALL THE CHANGES REQUESTED BY THE PREVIOUS REVIEWER BEFORE YOU RESUBMIT FOR ANOTHER REVIEW`
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @developertariq 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**